src/constants/index.js

```diff
@@ -1,5 +1,9 @@
 const CONSTANTS = {
   SIGNUP: 'SIGNUP',
+  RATE_ARTICLE: 'RATE_ARTICLE',
+  AVERAGE_RATING: 'AVERAGE_RATING',
+  RATING_ERROR: 'RATING_ERROR',
 };
 
 export default CONSTANTS;
+export const TOKEN = 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJlbWFpbCI6InRlc3QxQHRlc3QuY29tIiwiZXhwIjoxNTQxODgzNjU2fQ.WvUOntR_I3uV_l0sjmkg-WLA1XErlY6N6guycRS27Ek';
```

Comment:
Why is the token defined as a constant?
I thought it is generated dynamically every time a user logs in

Reply:
This should be changed once login has been implemented... I was just using it to test the functionality of this feature